### ISSUE TITLE
JAVA_OPTS Parameters

### ISSUE THREAD

**Issue by mateusz1364** [GoogleContainerTools/distroless#857]
In any way to working with JAVA_OPTS parameters. Example below?

```
FROM gcr.io/distroless/java:11
ADD target/app.jar /home/
WORKDIR /home/
ENV JAVA_OPTS=""
CMD ["$JAVA_OPTS app.jar"]
```

**Comment by loosebazooka** [comment]
Does this work with other base images? like [`adoptopenjdk/openjdk11`](https://hub.docker.com/r/adoptopenjdk/openjdk11)?

**Comment by mateusz1364** [comment]
I have to make a choice base image for mikroservices in much project (50 microservices+) and I need a quick response, sorry. I closed the issue. 

Thanks!

**Comment by loosebazooka** [comment]
FWIW, [JAVA_OPTS is a convention that is automatically picked up by the java runner](https://stackoverflow.com/questions/5241743/how-do-i-use-the-java-opts-environment-variable). So you do not need to include it in the cmd.

**Comment by chanseokoh** [comment]
> FWIW, [JAVA_OPTS is a convention that is automatically picked up by the java runner](https://stackoverflow.com/questions/5241743/how-do-i-use-the-java-opts-environment-variable). So you do not need to include it in the cmd.

No, it's picked up by _some_ servers (e.g., tomcat) and apps. Better use [`JAVA_TOOL_OPTIONS`](https://stackoverflow.com/a/40736821/1701388).
